src/xml/libxml2.cr

```diff
@@ -17,6 +17,15 @@ require "./save_options"
   {% end %}
 {% end %}
 lib LibXML
+   {% if (version = env("LIBXML_VERSION")) && (version.strip != "") %}
+     VERSION = {{env("LIBXML_VERSION")}}
+   {% elsif !flag?(:win32) || flag?(:gnu) %}
+     VERSION = {{`pkg-config libxml-2.0 --silence-errors --modversion 2> /dev/null || echo 2.9.0`.strip.stringify}}
+   {% else %}
+     # TODO: figure out the actual libxml version on *-windows-msvc target
+     VERSION = "2.9.0"
```

Comment:
libxml 2.9 was released in 2012. It's safe to assume this minimum version.

Reply:
MinGW fails to invoke the `pkg-config` command. Does it need to be wrapped in `sh -c "pkg-config ..."`?

Reply:
> libxml 2.9 was released in 2012. It's safe to assume this minimum version.

We should add this as a code comment.